### ISSUE TITLE
Wrong normalization on Kitti?

### ISSUE THREAD

**Issue by autosquid** [chenfengxu714/SqueezeSegV3#12]
Around https://github.com/chenfengxu714/SqueezeSegV3/blob/master/src/tasks/semantic/dataset/kitti/parser.py#L171:

```
proj = torch.cat([proj_range.unsqueeze(0).clone(),
                      proj_xyz.clone().permute(2,0,1),
                      proj_remission.unsqueeze(0).clone()])
proj = (proj - self.sensor_img_means[:, None, None]) / self.sensor_img_stds[:, None, None]
```

The order of channels in `proj` is (range, z, x, y, remission), but `sensor_image_means` is in the order of (range,x,y,z,signal).

So I guess here's something wrong?

**Comment by chenfengxu714** [comment]
Sorry for my late reply, I did't notice the issue recently. Yes, I made a small mistake here, while we also find the normalization on the KITTI doesn't have much influence and just slight drops the performance. It is true that right normalization with accurate mean and std are better. We will update this soon.
